Makefile

```diff
@@ -66,6 +66,19 @@ dep-build: ## Build the Helm chart with latest dependencies from the current Hel
 render: repo-update dep-build ## Render the Helm chart with the examples as input
 	examples/render-examples.sh || exit 1
 
+##@ Test
+# Tasks related to testing the Helm chart
+
+.PHONY: lint
```

Comment:
is `ct` installed as part of some make action also? or is setting up the tool out-of-band for local?

Reply:
Good catch. Originally I thought [chart testing (ct)](https://github.com/signalfx/splunk-otel-collector-chart/blob/18709a619c02fae9889a64472c0ec6950620b273/.github/workflows/lint-test.yaml#L22) was installed with helm, looks like it is not. Adding ct to the 'make install-tools' action.